buildscripts/github/find_workflow_runs.py

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/env python
+"""Find successful llvmlite workflow run IDs for a given git tag."""
+
+import argparse
+import json
+import os
+import subprocess
+import sys
+
+
+def find_workflow_run(workflow_name, tag, repo, token):
+    """
+    Find the latest successful workflow run for a given workflow and tag.
+
+    Args:
+        workflow_name: Name of the workflow file (e.g., 'llvmlite_conda_builder.yml')
+        tag: Git tag to search for
+        repo: Repository in format 'owner/repo' (e.g., 'numba/llvmlite')
+        token: GitHub token for API authentication
+
+    Returns:
+        Run ID as a string, or None if not found
+    """
+    print(f"Search for successful run of {workflow_name} on tag {tag} in {repo}", file=sys.stderr)
+
+    env = os.environ.copy()
+    env['GH_TOKEN'] = token
+
+    try:
+        result = subprocess.run(
+            [
+                "gh", "api",
+                "-H", "Accept: application/vnd.github+json",
+                "-H", "X-GitHub-Api-Version: 2022-11-28",
+                f"/repos/{repo}/actions/workflows/{workflow_name}/runs",
+                "--jq", f'.workflow_runs[] | select(.head_branch == "{tag}" and .conclusion == "success") | .id'
+            ],
+            capture_output=True,
+            text=True,
+            check=True,
+            env=env
+        )
+
+        run_ids = result.stdout.strip().split('\n')
+        if run_ids and run_ids[0]:
+            run_id = run_ids[0]
+            print(f"Found run ID {run_id} for {workflow_name}", file=sys.stderr)
+            return run_id
+
+        print(f"ERROR: No successful run found for {workflow_name} on tag {tag}", file=sys.stderr)
+        return None
+
+    except subprocess.CalledProcessError as e:
+        print(f"ERROR: Failed to query GitHub API: {e}", file=sys.stderr)
+        print(f"stderr: {e.stderr}", file=sys.stderr)
+        return None
+
+
+def main():
+    parser = argparse.ArgumentParser(
+        description='Find successful workflow run IDs for a given tag'
+    )
+    parser.add_argument(
+        '--tag',
+        default=os.environ.get('TAG', ''),
+        help='Git tag to search for (default: TAG env var)'
+    )
+    parser.add_argument(
+        '--repo',
+        default=os.environ.get('GITHUB_REPOSITORY', 'numba/llvmlite'),
```

Comment:
```suggestion
        default=os.environ.get('GITHUB_REPOSITORY'),
```
get rid of the default